api/ofdpa.proto

```diff
@@ -65,6 +65,12 @@ service OfdpaRpc {
 
   rpc ofdpaStgStatePortSet(openconfig_spanning_tree.Stp_Rstp_Interface_State) returns (OfdpaStatus) {}
 
+  rpc ofdpaTrunkCreate(trunk.Interface_Trunk_Description) returns (OfdpaStatus) {}
+  rpc ofdpaTrunkDelete(trunk.Interface_Trunk_Description) returns (OfdpaStatus) {}
+  rpc ofdpaTrunkPortPSCSet(trunk.Interface_Trunk_Description) returns (OfdpaStatus) {}
+
+  rpc ofdpaTrunkPortMemberSet(trunk.Interface_Trunk_Description) returns (OfdpaStatus) {}
```

Comment:
the actual call is named ofdpaPortTrunkGroupSet

Also it is weird that everything uses the same message if the actual ofdpa API calls take different arguments.

So this doesn't match the API of ofdpa at all.